3p/integration.js

```diff
@@ -502,6 +502,9 @@ function installContextUsingStandardImpl(win) {
     nextTick(win, () => cb([initialIntersection]));
     return unlisten;
   };
+  win.context.observePosition = cb => {
```

Comment:
can you just `win.context.observePosition = observePosition` or does the context/this preservation matter here?